dataframe-openapi/src/test/kotlin/OpenApiTests.kt

```diff
@@ -478,507 +479,609 @@ class OpenApiTests : JupyterReplTestCase() {
-        val code = execGeneratedCode(advancedExample, fullFunctionName.unquoted)
+        val code = openApi.readCodeForGeneration(
+            file = advancedExample,
+            name = fullFunctionName.unquoted,
+            extensionProperties = true,
+            generateHelperCompanionObject = false,
+        )
             .trimLines()
 
-        @Language("kt") val breedEnum = """
-            enum class Breed(override val value: kotlin.String) : org.jetbrains.kotlinx.dataframe.api.DataSchemaEnum {
-                DINGO("Dingo"),
-                HUSKY("Husky"),
-                RETRIEVER("Retriever"),
-                SHEPHERD("Shepherd");
-            }
-        """.trimLines()
-
-        code should haveSubstring(breedEnum)
-
-        @Language("kt") val dogInterface = """
-            @DataSchema(isOpen = false)
-            interface Dog : $functionName.Pet {
-                override val tag: kotlin.String
-                val bark: kotlin.Boolean?
-                val breed: $functionName.Breed
-                public companion object {
-                    public val keyValuePaths: kotlin.collections.List<org.jetbrains.kotlinx.dataframe.api.JsonPath>
+        @Language("kt") val s = """
+            interface `Other advanced test` {
+                
+                enum class EyeColor(override val value: kotlin.String) : org.jetbrains.kotlinx.dataframe.api.DataSchemaEnum {
+                    BLUE("Blue"),
+                    YELLOW("Yellow"),
+                    BROWN("Brown"),
+                    GREEN("Green");
+                }
+                @DataSchema(isOpen = false)
+                interface Pet {
+                    @ColumnName("pet_type")
+                    val petType: kotlin.String
+                    @ColumnName("value")
+                    val `value`: kotlin.Any?
+                    val name: kotlin.String
+                    val tag: kotlin.String?
+                    val other: kotlin.Any?
+                    @ColumnName("eye_color")
+                    val eyeColor: `Other advanced test`.EyeColor?
+                    public companion object {
+                      public val keyValuePaths: kotlin.collections.List<org.jetbrains.kotlinx.dataframe.api.JsonPath>
                         get() = listOf()
-
-                    public fun org.jetbrains.kotlinx.dataframe.DataFrame<*>.convertToDog(convertTo: org.jetbrains.kotlinx.dataframe.api.ConvertSchemaDsl<Dog>.() -> kotlin.Unit = {}): org.jetbrains.kotlinx.dataframe.DataFrame<Dog> = convertTo<Dog> { 
-                        convertDataRowsWithOpenApi() 
-                        convertTo()
+                
+                      public fun org.jetbrains.kotlinx.dataframe.DataFrame<*>.convertToPet(convertTo: org.jetbrains.kotlinx.dataframe.api.ConvertSchemaDsl<Pet>.() -> kotlin.Unit = {}): org.jetbrains.kotlinx.dataframe.DataFrame<Pet> = convertTo<Pet> { 
+                          convertDataRowsWithOpenApi() 
```

Comment:
Something wrong with formatting in the strings with code

Reply:
should haveSubstring doesn't show diff between two texts, and with generated code being 500 lines or so it's impossible to find why assertion fails.
so i refactored it to use shouldBe
In the editor code looks ok